meteor/client/lib/ReactMeteorData/ReactMeteorData.tsx

```diff
@@ -364,12 +365,12 @@ export function useSubscription<K extends keyof AllPubSubTypes>(
  */
 export function useSubscriptions<K extends keyof AllPubSubTypes>(
 	sub: K,
-	argsArray: Parameters<AllPubSubTypes[K]>[]
+	argsArray: Array<Parameters<AllPubSubTypes[K]> | undefined | null>
 ): boolean {
 	const [ready, setReady] = useState<boolean>(false)
 
 	useEffect(() => {
-		const subscriptions = Tracker.nonreactive(() => argsArray.map((args) => meteorSubscribe(sub, ...args)))
+		const subscriptions = Tracker.nonreactive(() => _.compact(argsArray).map((args) => meteorSubscribe(sub, ...args)))
```

Comment:
Instead of `_.compact(argsArray)`, you can use `argsArray.filter(Boolean)`, to avoid adding the underscore dependency

Reply:
That approach doesn't correct the typings, so isnt quite as simple a change to make 
![image](https://github.com/nrkno/sofie-core/assets/1327476/57a1eba2-e562-474b-a793-27268be3a1ba)